### PR TITLE
Fix / No import of generic T in utils #167

### LINKED ISSUE #167: No import of generic T in utils
Some of the sections use a previously created T and import it from `utils` instead of creating their own TypeVar. These references should be removed in favor of creating their own. In addition, ensure `T` is private if possible to reduce it's availability in imports.

### PR DESCRIPTION
**Pull request by TuTomasz**
### Issue

Fixes #167 

### Description

Removed existing T imports from utils substituted with local _T TypeVar in (scheduler.py). 
Made T private where applicable (data_store.py, serializable.py, utils.py).

### Testing
Code Review.

### Checklist
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] 🤔 **CONSIDER** adding a unit test if your PR resolves an issue.
- [x] ✅ **DO** check open PR's to avoid duplicates.
- [x] ✅ **DO** keep pull requests small so they can be easily reviewed.
- [x] ✅ **DO** build locally before pushing.
- [x] ✅ **DO** make sure tests pass.
- [x] ✅ **DO** make sure any new changes are documented.
- [x] ✅ **DO** make sure not to introduce any compiler warnings.
- [x] ❌**AVOID** breaking the continuous integration build.
- [x] ❌**AVOID** making significant changes to the overall architecture.

💚Thank you!

### CODE REVIEW

**Comment by keithrfung** [comment]
Thanks for your pull request! We'll be reviewing shortly.

File: src/electionguard/data_store.py [hunk 1]
```diff
@@ -12,18 +12,18 @@
 )
 
 
-T = TypeVar("T")
+_T = TypeVar("_T")
 U = TypeVar("U")
```
Comment (keithrfung) [inline src/electionguard/data_store.py:16]: Love this PR. Would you mind also doing this for `U` here in this file?

**Review by keithrfung — CHANGES_REQUESTED** [review]

File: src/electionguard/data_store.py (hunk 1 above)
Comment (TuTomasz) [reply src/electionguard/data_store.py]: Sure no problem. Made the changes and squashed it with the previous commit. Additionally made U private in one other file  utils.py for consistency.

**Review by keithrfung — APPROVED** [review]
lgtm. We're waiting to push in a couple other changes prior to merging.

**Review by AddressXception — APPROVED** [review]

**Pull request merged**